src/wolfcrypt/asn.py

```diff
@@ -0,0 +1,41 @@
+# asn.py
+#
+# Copyright (C) 2006-2020 wolfSSL Inc.
+#
+# This file is part of wolfSSL. (formerly known as CyaSSL)
+#
+# wolfSSL is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# wolfSSL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA
+
+# pylint: disable=no-member,no-name-in-module
+
+from wolfcrypt._ffi import ffi as _ffi
+from wolfcrypt._ffi import lib as _lib
+
+from wolfcrypt.exceptions import WolfCryptError
+
+if _lib.ASN_ENABLED:
+    def pem_to_der(file, pem_type):
```

Comment:
I'm not sure why it should get a file as argument, and not a buffer. It's a bit confusing to me since wc_PemToDer works on buffers.

Would it perhaps make more sense that the caller invokes `open`/`read` on the file, retrieves the pem buffer, then calls `pem_to_der()` on it?

Reply:
Good feedback. I agree.